pages/blog/index.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <nui-container class="pt-16">
+    <div class="flex flex-wrap justify-center mb-8">
+      <div class="text-center p-4 sm:p-0">
+        <h1 class="text-3xl xl:text-4xl text-nuxt-gray font-medium leading-normal mb-6 lg:pt-4">
+          NUXT<span class="text-nuxt-lightgreen">JS</span> Blog<br>
+        </h1>
+        <h3 class="xl:text-lg text-gray-600 font-medium leading-relaxed mb-6">
+          Discover articles from the core team and contributors about NuxtJS, tips and tricks included!
+        </h3>
+      </div>
+    </div>
+    <section class="post-feed">
+      <article v-for="(post, p) in posts" :key="p" class="post-card-container hover:shadow-nuxt">
+        <nuxt-link :to="{ name: 'blog-slug', params: { slug: post.slug }}" tag="div" class="post-card">
```

Comment:
For SEO purposes article card should have actual link to the article (internal linking), so I suggest to leave `nuxt-link` as `a` tag, or put `a` somewhere inside (for example wrap title)